### PR TITLE
Fixing spa bugs with sln file generation

### PR DESCRIPTION
**Pull request by VAlmea**
## Related to Issue
Fixes #

#205 
#204 
#203
#202

## How Has This Been Tested?
I created a new project for each spa template available option 
The project was added automatically to the solution and the sln and csproj files store in the proper folder 
I successfully compiled each project in debug and release mode and test the module in a DNN blank solution 

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (WillStrohl) [inline .vscode/launch.json:1]: @VAlmea Should this file be included, or should the `.vscode` folder be ignored?

**Review by WillStrohl — COMMENTED** [review]
@VAlmea This looks good, except for the one question I send in this review.  :)

File: .vscode/launch.json (hunk 1 above)
Comment (VAlmea) [reply .vscode/launch.json]: This file is used to debug the generator in development mode. Maybe it would be a better idea to update the documentation

Comment (WillStrohl) [reply .vscode/launch.json]: Yes. We shouldn't commit it, and instead, create documentation to tell people how to do this themselves.  We'll create a page in the Wiki for this.

**Review by WillStrohl — APPROVED** [review]
Looks good.

**Pull request merged**